shairport-sync-metadata-reader.c

```diff
@@ -252,6 +253,12 @@ int main(void) {
             break;
           case 'PICT':
             printf("Picture received, length %u bytes.\n",length);
```

Comment:
Add TODO: remove old pictures

Reply:
Currently, old pictures are removed whenever you start up a new instance of the same source (i.e. putting Jason's iPhone on channel 3 will remove all prior pictures in channel 3's directory). I will admit that this solution kind of sucks though

Reply:
Just thought of another way to handle picture deletion - if iTunes pauses for about 10 seconds, the 'PAUSED' information gets updated in 'currentSong'.
Maybe the files can be deleted at this time? Still isn't perfect, but it would be better than the current implementation.